### ISSUE TITLE
[BUG] unrealible search clause

### ISSUE THREAD

**Issue by plonknimbuzz** [nacasha/CIgniter-Datatables#5]
datatables searching is using `OR` not `AND`

https://github.com/nacasha/CIgniter-Datatables/blob/261f83c186b7657dfb9894ce08a0f1c6b5279e58/libraries/Datatables.php#L231-L237

this bug make me cant display column which have null value.

should be: 
```
if($search != "") {
	$i = 0;
	foreach($this->searchable as $sCol){
		if($i==0) $this->_db->like($sCol, $search);
		else $this->_db->or_like($sCol, $search);
		$i++;
	}
}
```
or
```
if($search != "") {
	for($i=0; $i< count($this->searchable);$i++){
		if($i==0) $this->_db->like($this->searchable[$i], $search);
		else $this->_db->or_like($this->searchable[$i], $search);
	}
}
```
note: `for` is more fast than `foreach`

**Comment by nacasha** [comment]
Thanks for you report, I will fix this issues asap :wink:

**Comment by nacasha** [comment]
fixed 1631892
